saleor/graphql/order/resolvers.py

```diff
@@ -58,3 +59,14 @@ def resolve_order_by_token(token):
         .filter(token=token)
         .first()
     )
+
+
+# TODO: We should optimize it in/after PR#5819
+def resolve_order_shipping_methods(root: models.Order, info):
+    available = get_valid_shipping_methods_for_order(root)
+    if available is None:
```

Comment:
Does this still return None?

Reply:
it doesn't, empty lists will be returned from `get_valid_shipping_methods_for_order` 👍